lib/plugins/features/movies/index.js

```diff
@@ -32,6 +33,19 @@ exports.register = (server, options, next) => {
     }
   }]);
 
+  server.route([{
+    method: 'POST',
+    path: '/movies/{id}/locations',
+    config: {
+      handler: (request, reply) => {
+        reply(Controller.assignLocation(request.params.id, request.payload.location_id));
+      },
+      validate: {
+        params: LocationToMovieValidator
```

Comment:
Nit: You could also add a `payload` validator. Since both validators are relatively simple, you can define them in-line. See [this example in API](https://github.com/lob/lob-api/blob/master/src/plugins/features/areas/index.js#L91-L93).